Assets/Scripts/WorldEngine/World.cs

```diff
@@ -3323,151 +3313,6 @@ private void GenerateTerrainRainfall(TerrainCell cell, bool justSetRainfallSourc
         CalculateAndSetRainfall(cell, rainfallValue);
     }
 
-    private void GenerateTerrainRainfall2(TerrainCell cell, bool justSetRainfallSources = false, bool setDependencies = true)
```

Comment:
Why was this removed?

Reply:
This was an experimental function that was supposed to replace GenerateTerrainRainfall. But it was very slow and I didn't like the results. I kept the function because I wanted to rework it and make it better. But now the plan is to eventually introduce wind patterns, which will force a rewrite of rainfall generation in a completely different way. Therefore I don't want to keep a function that will never be used.